src/coreclr/debug/daccess/daccess.cpp

```diff
@@ -4324,7 +4324,7 @@ ClrDataAccess::TranslateExceptionRecordToNotification(
     GcEvtArgs pubGcEvtArgs = {};
     ULONG32 notifyType = 0;
     DWORD catcherNativeOffset = 0;
-    TADDR nativeCodeLocation = NULL;
+    TADDR nativeCodeLocation = (TADDR)NULL;
```

Comment:
Since you have found all of these... thoughts on doing a `TADDR::Null` static? I am making this suggestion based on the assumption that we going to stuck with this `NULL` casting for `TADDR` for a long time.

Reply:
`TADDR` is a typedef from `uintptr_t` so we can't do anything like that today.

In the future, I'd like to move in that direction (by making `TADDR` an enum-class or something like that to get us more explicit typing without regressions on Windows) but that's going to be a lot more work than I want to do in this PR.

Reply:
Oh, right. I was thinking it was a `struct`. All good.